assets/src/pages/room/components/MediaControlButtons.tsx

```diff
@@ -366,7 +366,7 @@ const MediaControlButtons: FC<Props> = (props: Props) => {
     <div>
       <div
         onClick={toggleShow}
-        className="absolute left-1/2 top-[-15px] z-[-10] h-[15px] w-[50px] -translate-x-1/2 rounded-t-lg hover:bg-gray-700 hover:bg-opacity-90"
+        className="absolute left-1/2 top-[-15px] z-[-10] h-[15px] w-[50px] -translate-x-1/2 rounded-t-lg hover:bg-brand-dark-gray/90 hover:bg-opacity-90"
```

Comment:
Since you've added `/90` opacity, you should remove the old class
```suggestion
        className="absolute left-1/2 top-[-15px] z-[-10] h-[15px] w-[50px] -translate-x-1/2 rounded-t-lg hover:bg-brand-dark-gray/90"
```